libexec/bats

```diff
@@ -77,7 +77,7 @@ abs_dirname '.' 'BATS_CWD'
 
 export BATS_PREFIX
 export BATS_CWD
-export BATS_TEST_PATTERN='^ *@test +(.+) +\{ *(.*)$'
+export BATS_TEST_PATTERN="^[[:space:]]*@test[[:space:]]+(.*[^[:space:]])[[:space:]]+\{(.*)\$"
```

Comment:
Aren't the extra wrapping brackets around `[:space:]` unnecessary? (Except in the single negation instance?)

Reply:
Apparently they _are_ required. TIL (I thought they operated just like the shortcut forms.)

Reply:
They're all required. Just tried locally to make sure, but the outside brackets just announce a set of characters to match, and `[:space:]` is special syntax announcing a character class within those outer brackets.